Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AddArchiveTask.java

```diff
@@ -201,10 +201,21 @@ public void run() {
                         success = true;
                         newDataSources.addAll(internalDataSource.getContent());
                         
-                        // Update the names for all new data sources to be the root archive plus the name of the data source
+                        // update data source info
                         for (Content c:internalDataSource.getContent()) {
                             if (c instanceof DataSource) {
                                 DataSource ds = (DataSource) c;
+                                 
+                                // Read existing aquisition details and update them
+                                String details = "Extracted from archive: " + archivePath.toString();
```

Comment:
I find this text confusing. See JIRA for details.

Reply:
@dgrove727 see my reply in JIRA.